test/common/README.md

```diff
@@ -194,7 +194,12 @@ Indicates whether OpenSSL is available.
 ### hasFipsCrypto
 * [&lt;boolean>]
 
-Indicates `hasCrypto` and `crypto` with fips.
+Indicates that Node.js has been linked with a FIPS compatible OpenSSL library,
+and that FIPS as been enabled using `--enable-fips`.
+
+To only detect if the OpenSSL library is FIPS compatible, regardless if it has
+been enabled or not, then `process.config.variables.openssl_is_fips` can be
+used to determine that situation.
```

Comment:
One very important challenge with using `process.config` is that there are userland modules that overwrite it's value. That is, it cannot be relied upon in every case.

Reply:
This is the documentation for our test utility functions. Interference from userland modules shouldn't be an issue.